config/initializers/assets.rb.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += ['payments.js']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.